desc/compute/_equil.py

```diff
@@ -318,28 +448,28 @@ def _F_theta(params, transforms, profiles, data, **kwargs):
     transforms={},
     profiles=[],
     coordinates="rtz",
-    data=["sqrt(g)", "B^theta", "J^rho"],
+    data=["B^theta", "F_helical"],
 )
 def _F_zeta(params, transforms, profiles, data, **kwargs):
-    data["F_zeta"] = data["sqrt(g)"] * data["B^theta"] * data["J^rho"]
+    data["F_zeta"] = data["B^theta"] * data["F_helical"]
     return data
 
 
 @register_compute_fun(
     name="F_helical",
     label="F_{helical}",
-    units="N \\cdot m^{-2}",
-    units_long="Newtons / square meter",
+    units="N / (T m)",
+    units_long="Newtons / (Tesla meters)",
```

Comment:
can you explain this change a bit? Why not keep it in units of Newtons?

Reply:
$N / (T \cdot m)$ equals amps which is $F_{\text{helical}} = T \cdot m / \mu_0$

Reply:
The old units of $N / m^2$ weren't correct

Reply:
hmm ok, I thought we had all the `F_` stuff in units of Newtons or Newtons/meter^2. Could we refactor things to keep that? (i think that was the idea behind `e_helical`?)

Reply:
The covariant components of $F$ are the ones already in $N / m^2$. (e.g. $F_{\zeta} = B^{\theta} F_{\text{helical}}$). Daniel's original paper uses this quantity to decompose $F = F_{\rho} \nabla \rho + F_{\beta} \beta$ where $F_{\beta} =F_{\text{helical}}$. I'm hesitant to change this because the units are consistent with what we expect: $F = N m^{-3} =  N m^{-2} * m^{-1} + N / (T \cdot m) * T m^{-2}$

Reply:
(from [equation 11a](https://github.com/PlasmaControl/DESC/blob/add_all_limits/publications/dudt2020/dudt2020desc.pdf))

Reply:
Yeah I remember at some point we changed the definition of the helical basis vector so that both `F_helical` and `F_rho` would have the same units. @ddudt do you remember the details?

Reply:
OK, looks like before v0.5.0 we computed $F_{helical} = \sqrt{g} J^\rho B^\zeta$ and $\mathbf{e}_{helical} = \mathbf{e}^\theta - \iota \mathbf{e}^\zeta$ but that was technically incorrect, because $\theta$ isn't the straight field line coordinate, so $\iota \neq B^\theta/B^\zeta$.

I think it would still be cleaner to factor out $B^\zeta$ from $\mathbf{e}^{helical} = \mathbf{e}^\theta - B^\theta/B^\zeta \mathbf{e}^\zeta$ and add it to $F_{helical} = \sqrt{g} J^\rho B^\zeta$ to make the force term have the same units as the others, and the basis vector to have the correct units of $m^{-1}$. We should probably also rename `e_helical` to `e^helical` since its really a contravariant vector.

Reply:
could be done in a new PR though, doesn't have to be here.

Reply:
The master branch matches my original paper, and I think we should keep it that way. If I recall correctly, we had changed `F_helical` to have the same units as `F_rho` at one point, but then changed it back because it was incorrect. We can't factor out $B^\zeta$ from $e^{helical}$ because $\theta$ is not a straight field line coordinate. I don't think it's a big deal because the user will rarely use `F_helical` directly. 

Thanks for catching this unit mistake @unalmis, but I would prefer to say Amperes instead of "Newtons / (Tesla meter)" for simplicity. Also @f0uriest I think it is already correctly named `e^helical`.